src/index.html

```diff
@@ -1,13 +1,367 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="shortcut icon" type="image/ico" href="./images/icons/favicon-logo.ico"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <script src="./scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="nav-bar">
+            <a class="logo" href="./index.html">
+              <img
+                class="logo__image"
+                src="./images/icons/logo.svg"
+                alt="bose"
+              >
+            </a>
+            <div class="nav-bar__link">
+              <a
+                class="icon icon--call nav-bar__link--call header__call-link" href="tel:+1 234 555-55-55"
+              ></a>
+              <a
+                class="icon icon--menu nav-bar__link--menu-opener" href="#menu"
+              ></a>
+            </div>
+          </div>
+          <h1 class="header__title">
+            The world shades.
+            <br>
+            Your music shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
```

Comment:
https://prnt.sc/WOXm6aLjm_1L => you sholudn't have scroll with menu, fix it with js script

Reply:
i'm not shure that is in tech task....